corgidrp/data.py

```diff
@@ -724,11 +724,110 @@ def copy(self, copy_data = True):
 
         return new_bp
 
+class DetectorParams(Image):
+    """
+    Class containing detector parameters that may change over time
```

Comment:
It would be good to state here how this should be used. It looks like like a user will read keywords from the extension header, but it would be good to lay that out.

Reply:
Oh, or is it to use the .params dictionary?